k8s/eks-setup.md

```diff
@@ -0,0 +1,230 @@
+
+```
+helm install zqd-test-1 charts/zqd \
+  --set AWSRegion="us-east-2" \
+  --set datauri="s3://zqd-demo/mark/zqd-meta" \
+  --set image.repository="123456789012.dkr.ecr.us-east-2.amazonaws.com/" \
+  --set useCredSecret=false
+```
+
+## Exposing the zqd endpoint
+
+To access the running zqd instance, use kubectl port-forward:
+```
+kubectl port-forward svc/zqd-test-1 9867:9867 &
+```
+There is a script `k8s/zqd-port.sh` that does this after removing any existing port-forwards.
+
+## Using a K8s hosted zqd to search large logs in the cloud
+This decribes a set of steps to use `zar import` to process large log files that can then be searched with the zqd you just deployed. In order to do this, you will need an EC2 instance in the same region as your EKS cluster. Follow the steps below to set this up.
+
+## Setting up an EC2 instance to run zar import
+
+Create an EC2 instance in the same region as the S3 buckets you want to access. SSH into the instance, and:
+```
+# provide your credentials
+aws configure
+# make sure you can see the S3 buckets
+aws s3 ls
+# install golang tools from https://golang.org/dl/
+wget https://golang.org/dl/go1.14.7.linux-amd64.tar.gz
+sudo tar -C /usr/local -xzf go1.14.7.linux-amd64.tar.gz
+echo "export PATH=$PATH:/usr/local/go/bin" >> ~/.bash_profile
+source ~/.bash_profile
+go version  # make sure go is there!
+# install git, clone and install zq
+sudo yum install git -y
+git clone https://github.com/brimsec/zq
+cd zq
+make install
+~/go/bin/zar help  # make sure zar is built!
+```
+Now you have an environment where you can run zar and it can operate on data in S3 with good bandwidth. You can update it as needed with:
+```
+cd zq
+git pull
+make install
+```
+You may want to use an SSD for $TMPDIR for zar import where the sort can spill to disk. Consider this if zar runs out of space for sorting.
+
+First use `lsblk` to find an SSD device, then mount it similarly to the following example:
+```
+lsblk  # look for a free device, e.g. nvme1n1
+sudo mkfs -t xfs /dev/nvme1n1
+sudo mkdir /data
+sudo mount /dev/nvme1n1 /data
+df  # Verify that /data is mounted
+sudo chown ec2-user /data  # so zar can write to it
+echo "export TMPDIR=/data" >> ~/.bash_profile
+source ~/.bash_profile
+```
+
+## Using zar import for large S3 logs
+In the following examples, we import some log files that are already present in S3 in zeek format, and create a local archive for use by zqd. The following commands should be run on an ec2 instance in the same region as your EKS cluster:
+```
+# this is a 7 MB log -- takes a few seconds
+time ~/go/bin/zar import -R s3://zqd-demo-1/mark/zeek-logs/dpd s3://brim-sampledata/wrccdc/zeek-logs/dpd.log.gz
+
+# and a 267 MB log -- takes a few minutes
+time ~/go/bin/zar import -R s3://zqd-demo-1/mark/zeek-logs/dns s3://brim-sampledata/wrccdc/zeek-logs/dns.log.gz
+
+# and a 14 GB log -- takes about two hours, and will fail without TMPDIR space
```

Comment:
Hopefully this will be a lot faster once #1397 is merged.

Reply:
I am moving this to troubleshooting.md also, and removing the timing estimates